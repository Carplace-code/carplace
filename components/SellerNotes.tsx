/* eslint-disable */

export default function SellerNotes({ listing }: { listing: any }) {
  return (
    <div className="rounded-md border bg-white p-6">
      <h2 className="mb-4 text-lg font-semibold">Seller Notes</h2>
      <p className="text-sm text-gray-700">{listing.description || "No description provided by seller."}</p>
    </div>
  );
}
