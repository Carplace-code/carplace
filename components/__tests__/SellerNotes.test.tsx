// components/__tests__/SellerNotes.test.tsx
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

import SellerNotes from "../SellerNotes";

describe("SellerNotes", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<SellerNotes />);
    expect(container).toMatchSnapshot();
  });
});
