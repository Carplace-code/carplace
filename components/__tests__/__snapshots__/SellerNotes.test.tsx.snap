// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SellerNotes > renders correctly and matches snapshot 1`] = `
<div>
  <div
    class="rounded-md border bg-white p-6"
  >
    <h2
      class="mb-4 text-lg font-semibold"
    >
      Seller Notes
    </h2>
    <p
      class="text-sm text-gray-700"
    >
      Excelente estado general, mantenido en taller oficial. Inspección técnica al día, sin siniestros reportados. Interior como nuevo, se entrega con mantención recién hecha. Llantas originales y neumáticos nuevos.
    </p>
  </div>
</div>
`;
